### ISSUE TITLE
Error / crash on exit

### ISSUE THREAD

**Issue by jetXS** [TouchDesigner/TouchEngine-UE4#100]
![image](https://user-images.githubusercontent.com/10253645/138464870-91db4cb3-4c35-4651-96ac-b5de94186b21.png)
![image](https://user-images.githubusercontent.com/10253645/138465026-f66cf663-879c-466e-aa1a-16dbc374a3c4.png)

When closing UE while debugger is running. To be expected?

**Comment by ddthejj** [comment]
Fixed in 49573792089367117482d50930f9cb43b5f29c72

**Comment by jetXS** [comment]
tested and confirmed as fixed

**Comment by jetXS** [comment]
Reopening, had this one occurring a couple of times:

When exiting vanilla project with only plugin added and TouchEngine Actor added to world, no debugger attached

```
Unhandled Exception: 0xc06d007e

KERNELBASE
UE4Editor_TouchEngine!__delayLoadHelper2() [d:\a01\_work\6\s\src\vctools\delayimp\delayhlp.cpp:312]
UE4Editor_TouchEngine!_tailMerge_touchengine_dll()
UE4Editor_TouchEngine!UTouchEngine::~UTouchEngine() [D:\_DERIVATIVE\Github\UE_Tests\BlankXR\Plugins\TouchEngine-UE4\Source\TouchEngine\Private\UTouchEngine.cpp:891]
UE4Editor_TouchEngine!UTouchEngine::`vector deleting destructor'()
UE4Editor_CoreUObject
UE4Editor_CoreUObject
UE4Editor_CoreUObject
UE4Editor_CoreUObject
UE4Editor_CoreUObject
UE4Editor
UE4Editor
UE4Editor
UE4Editor
UE4Editor
UE4Editor
UE4Editor
UE4Editor
kernel32
ntdll
```

**Comment by ddthejj** [comment]
This should be fixed in e90b224d4889b92ffb965026eb692e415b807fa6

**Comment by jetXS** [comment]
Tested and could not reproduce, closing. Thanks.
